text/0037-disjoint-groups/0037-disjoint-groups.md

```diff
@@ -91,36 +91,24 @@ To avoid repeated splitting and merging due to small fluctuations in group size,
 
 ### Message routing
 
-To relay a message from an individual node on a given `route` in a node `n` in group `G(p)` for a destination _node_ `d`:
+When learning about a change in any group (new member, group split or merge), each node signs the sorted new list of members in _that_ group and sends it to each member of _its own_ group, so every node has a signature from each of its group peers, for the list of current members of each neighbouring group.
```

Comment:
Good point. I named the groups `A` and `B`. It's not exactly poetry now, but maybe less confusing. :grin: